recipes/isl/all/conanfile.py

```diff
@@ -1,10 +1,13 @@
-from conans import AutoToolsBuildEnvironment, ConanFile, tools
-from conans.errors import ConanInvalidConfiguration
-from contextlib import contextmanager
+from conan import ConanFile
+from conan.tools.files import chdir, copy, get, rmdir, apply_conandata_patches
```

Comment:
```suggestion
from conan.tools.files import chdir, copy, get, rmdir
```

Reply:
Back ported fix from 0.25 so now apply_conandata_patches is needed.